src/common/snippets/src/pass/transform_convert.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/remarks.hpp"
+#include <snippets/itt.hpp>
+
+#include "snippets/pass/transform_convert.hpp"
+#include "snippets/snippets_isa.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::snippets::pass::TransformConvertToConvertTruncation::TransformConvertToConvertTruncation() {
+    MATCHER_SCOPE(TransformConvertToConvertTruncation);
+    register_matcher(std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<ngraph::opset1::Convert>(), matcher_name),
```

Comment:
It may also match to ConvertSaturation, so maybe worth to add a check like on L41.
I would also check that we don't try convert truncation to truncation (to avoid useless work):
`is_type<ngraph::opset1::Convert>(n) && !(is_type<op::ConvertSaturation>(n) || is_type<op::ConvertTruncation>(n))`